README.md

```diff
@@ -11,6 +11,11 @@ make build
 ./harbor_exporter [flags]
 ```
 
+To build a Docker image:
+
+```
+docker build . --tag <mytag>
```

Comment:
I do have a make target `make dockerbuild`

Reply:
overlooked that, then that can be added to the readme